lineapy/plugins/taskgen.py

```diff
@@ -215,7 +215,7 @@ def get_tmpdirpickle_setup_task_definition(pipeline_name):
         pipeline_name=pipeline_name
     )
     return TaskDefinition(
-        function_name="dag_setup",
+        function_name="setup",
```

Comment:
Renamed it so that the snapshot doesn't change. It used to be that for these setup and teardown tasks that the "task_name" which is the key in the task_defs dictionary was not the same as "task_def.function_name". This is mostly to fix that. 

Orchestrators dont look for either name in particular.